airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/schemas/GET_FBA_REIMBURSEMENTS_DATA.json

```diff
@@ -0,0 +1,116 @@
+{
+  "title" : "FBA Reimbursements Data",
+  "description" : "FBA Reimbursements Data Reports",
+  "type" : "object",
```

Comment:
additionalProperties: true

Reply:
@marcosmarxm - I've updated the code. I thought it was covered by the code here: https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/spec.py#L13